magento/pwa-studio
# Cart page product links.

## Description

Added product links to the product listings on the cart page.

## Related Issue
Closes [PWA-740](https://jira.corp.magento.com/browse/PWA-740)

### Verification Stakeholders
@jimbo 
@soumya-ashok 
@schensley 

### Verification Steps
1. Go to the deployed app and add an item to the cart.
2. Go to the cart page and click on the item in the cart. It should take you to the product's PDP page.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
